src/init.cpp

```diff
@@ -749,7 +750,8 @@ static bool AppInitServers()
     RPCServer::OnStopped(&OnRPCStopped);
     if (!InitHTTPServer())
         return false;
-    StartRPC();
+    if (gArgs.GetBoolArg("-rpcallowstop", true)) StartRPC();
+    else StartRPC(false);
```

Comment:
`if ... else ...` is confusing.
Is `then` missed or it should be instead of `else`?